### DISCUSSION TITLE
Homelessness..sucks but Integrity..wins

### DISCUSSION THREAD

**Discussion by joemddr** [vercel/next.js#43726] [Show and tell]
Breach_coin_Environment @filed branding patented cryptocurrencies/token
“Secure within “
Thanks for reading or sharing your experiences personally with out having a team of friends, entrepreneurs, skilled in every aspect of the industry we are currently living in. Some are wondering if this is a scam as well. But I assure you it’s only a thought or way of thinking outside the box. What if our digital identity was exploited or taken away by someone who used either for monetary gain or other harmful intentions.. How would we prove that we were incorrectly blamed for a criminal or purposeful act.. or what we lost? Or be blamed in a middle man scenario.. some used to look at the amount of money or digits in a bank account or credit card amounts .. well that went downhill quick.. I’m sure there is a missing fundamental insurance companies use for proof or evidence of todays criminal activity or-even those who meant well like a GoFund me.. “we are not affiliated with the company Go fund me or work for any other organization that’s involved with them.. we didn’t get there approved disclosure to say there name or use in a comparison or mean to cause loss or damage to the name or brand.. since when did the freedom of expression or blunt statements become a lawsuit? Or some other paragraph of disclosures needed to advertise any kind of information to help people or protect them. What if I told you the fact you are reading this wasn’t by luck it’s a smart contract and by reading it your liable for loss of identifiable value. Our thought about that is the same.. redistribution of information rather copy or paste from a human being or version of a pc should always be the same.. expressing yourself about that is just as important.. that way we learn differently from each other.. what if the process from the power button to the power supply was a branded owned identifier as it technically is.. it’s a job intent to push button button x power supply y the info the pc gets is what we program it to be.. or maybe we don’t have enough data for a job.. we go to our phone learn about it.. well sometimes we take for granite what we do maybe intentionally or maybe purposefully.. I’ve monitored a lot of data logging in my career there’s a lot of information that needs to be taken in to diagnose failure.. we find it isolate it and repair it.. why can we not isolate our own bodies as an individual entity.. big businesses and governments thrive on our collective data to buy or distribute a product or service “let’s say” a global pandemic.. amounts of money again is not the purpose of this post.. we print money we mine coin we sell it for money money gets stolen money is it isn’t able to be collected or saved.: we lose our jobs we can’t feed our families or buy gas bc of inflation of course money doesn’t buy us consciousness or smarts.. we have to restructure processes to make a difference or change.. Breach coin isn’t worth money it’s a collection of data of the owners decision of input.. social environment.. if you choose to enroll in tokenized information transfer technology or not is your choice.. technically it’s already available for purchase.. as in “do not sell my information “ hyper links at bottoms of pages on whoever’s internet we chose that day.. again profitable business for the company who is behind that scam.. no state or government regulation or disclosures of self personalization of there own transaction of information should be taken for granite or sold without your permission.. permission isn’t required to have a conversation with your neighbor or friends.. but it’s exploited and sold or used as a tool for gain or benefit.. if you back me on my thought about a personalized token “breach-coin” I feel will change the game.. it’ needs a team of people to attempt changing the way we interact or transfer information.. proprietary information is important even if it’s a invite to a wedding or a lemonade stand . It will fill all loss space in a stock market.. population will be inside of the info not sold out to.. if your inside the box your daily activities are already monitored and than sold at our discretion of how you choose to profit from if you borrow or owe this will eliminate risk factoring by losing your way to purchase or pay or gift anyone or needs.. we won’t lose state money by bad actors stealing our information or will ever have to worry about identity theft or voting for our next leaders of community from prom queen king to county and state delegates.. it will eliminate worry for factors of loss we all worry about or pay extra for our pc or phone to have extra protection.. it’s not a prediction of what they or conspiracy to gain global control or power it’s the Next step.. thought.. for every person to benefit from.. in no way will we see a change without a attempt of personal choice to log in to the site we eventually choose to be part of a community of people and professionals, religious leaders or even the people. “Hackers” we put out of business or the greedy businessman or organizations that meant well but lost everything they really did work for.. Internet companies or even IRS .. they will be the first ones who recommend this secure service that we use everyday intent to prove our existence and worth and purpose.. not just identifying the person know the persons intent.. utilizing Ai technology that’s already available or used … Breach Coin gets information _ binds=firstname middlename lastname ssn dob dl_number dl_state gendermilitary_active amount_requested residence_type residence_lengthaddress1 address2 city state zip phone_home phone_cell contact_timeemail ip_addr pay_frequency net_income first_paydaysecond_payday employment_status employer_name job_title hire_datephone_work phone_work2 bank_name account_type direct_depositrouting_no account_no reference1_firstname reference1_lastnamereference1_relationship phone_reference1 reference2_firstnamereference2_lastname reference2_relationship phone_reference2 process_date…

Again this is only a expression of thought of choice for a change I feel is necessary..

Sent from my iPhone

**Comment by sooswastaken** [comment]
what the-
